crates/vim/src/motion.rs

```diff
@@ -783,6 +784,30 @@ impl Motion {
                     *selection.end.column_mut() += 1;
                 }
             }
+            Some(selection.start..selection.end)
+        } else {
+            None
+        }
+    }
+
+    // Expands a selection using self motion for an operator
```

Comment:
What does `self motion` refer to here? `self.motion`? Or something else?

Reply:
> What does `self motion` refer to here? `self.motion`? Or something else?

It should just be self here